src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -242,4 +289,53 @@ export class ServiceDaemonListComponent implements OnInit, OnChanges, AfterViewI
   trackByFn(_index: any, item: any) {
     return item.created;
   }
+
+  updateSelection(selection: CdTableSelection) {
+    this.selection = selection;
+  }
+
+  daemonAction(actionType: string) {
+    this.daemonService
+      .action(this.selection.first()?.daemon_name, actionType)
+      .pipe(take(1))
+      .subscribe({
+        next: (resp) => {
+          this.notificationService.show(
+            NotificationType.success,
+            'Daemon action scheduled',
```

Comment:
I think its good to tell what action is performing in the heading of the notification too
```suggestion
            'Daemon {{ actionType }} scheduled',
```